index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>"판다 마켓"</title>
```

Comment:
따옴표는 제거되는게 좋지 않을까 생각해요!
```suggestion
  <title>판다 마켓</title>
```